packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -50,6 +51,10 @@ function FeedNav(): ReactElement {
     [0, 1],
     DEFAULT_ALGORITHM_INDEX,
   );
+  const scrollClassName = useScrollTopClassName({
+    scrolledClassName: 'bg-transparent',
```

Comment:
This should affect everything already?

Reply:
What do you mean by affect everything already? :)

Reply:
I mean should this affect everything even when your not in experiment?

Reply:
It does not have any effect, since there's nothing behind the navbar. Once you scroll, the default bg will kick in. But if we want to be super safe, I can add it behind the experiment.